a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java

```diff
@@ -63,7 +63,7 @@ public static void setUpStatic() {
 	}
 	
 	@Before
-	public void setUp() throws Exception {
+	public void setUp() throws Exception {  
```

Comment:
It's a minor thing, but you should make sure trailing spaces are not added.

Reply:
I have updated as per discussion